tower/src/load_shed/error.rs

```diff
@@ -12,7 +12,8 @@ pub struct Overloaded {
 }
 
 impl Overloaded {
-    pub(crate) fn new() -> Self {
+    /// Construct a new overloaded error
+    pub fn new() -> Self {
```

Comment:
tiny style nit: if we're going to have a public 0-argument `new`, we may as well also implement `Default`...

Reply:
Updated, thanks for reviewing.